src/nrnoc/treeset.cpp

```diff
@@ -1887,20 +1891,21 @@ extern "C" void nrn_complain(double* pp) {
     hoc_Item* qsec;
     int j;
     Prop* p;
-    ForAllSections(sec) for (j = 0; j < sec->nnode; ++j) {
-        nd = sec->pnode[j];
-        for (p = nd->prop; p; p = p->next) {
-            if (p->param == pp) {
-                fprintf(stderr,
-                        "Error at section location %s(%g)\n",
-                        secname(sec),
-                        nrn_arc_position(sec, nd));
-                return;
+    ForAllSections(sec) {
+        for (j = 0; j < sec->nnode; ++j) {
+            nd = sec->pnode[j];
+            for (p = nd->prop; p; p = p->next) {
+                if (p->param == pp) {
+                    fprintf(stderr,
+                            "Error at section location %s(%g)\n",
+                            secname(sec),
+                            nrn_arc_position(sec, nd));
+                    return;
+                }
             }
         }
     }
-}
-fprintf(stderr, "Don't know the location of params at %p\n", pp);
+    End_ForAllSections fprintf(stderr, "Don't know the location of params at %p\n", pp);
```

Comment:
This line seems a bit unfortunate, but it's not the end of the world. Perhaps we can teach `clang-format` about the `End_ForAllSections` macro?

Reply:
@olupton : would that be just adding line here https://github.com/neuronsimulator/nrn/blob/121d968e764e292e5aa821cedfb5e9a8a7ed057d/.clang-format.changes#L3?

Reply:
I think so, yes

Reply:
Done.